src/typescript/sdk/src/emojicoin_dot_fun/emojicoin-dot-fun.ts

```diff
@@ -292,6 +293,42 @@ export class RegisterMarket extends EntryFunctionPayloadBuilder {
     this.feePayer = feePayer !== undefined ? AccountAddress.from(feePayer) : undefined;
   }
 
+  static async getGasCost(args: {
+    aptosConfig: AptosConfig;
+    registrant: AccountAddressInput; // &signer
+    registrantPubKey: PublicKey;
+    emojis: Array<HexInput>; // vector<vector<u8>>
+  }): Promise<{ data: { amount: number; unitPrice: number }; error: boolean }> {
+    const { aptosConfig } = args;
+    const payloadBuilder = new this({
+      ...args,
+      integrator: "0x0000000000000000000000000000000000000000000000000000000000000001",
```

Comment:
```suggestion
      integrator: AccountAddress.ONE,
```

I believe this should work, a bit more readable